contrib/completion/_fd

```diff
@@ -0,0 +1,210 @@
+#compdef fd
+
+##
+# zsh completion function for ripgrep
+#
+# Originally based on code from the zsh-users project — see copyright notice
+# below.
+
+autoload -U is-at-least
+
+_fd() {
+  local no='!' ret=1
+  local -a _arguments_options fd_file_types fd_args
+
+  if is-at-least 5.2; then
+      _arguments_options=( -s -S )
```

Comment:
Indentation here and below is inconsistent with rest of file

Reply:
Fixed.